profiles/desc/python_single_target.desc

```diff
@@ -7,4 +7,5 @@ python3_10 - Build for Python 3.10 only
 python3_11 - Build for Python 3.11 only
 python3_12 - Build for Python 3.12 only
 python3_13 - Build for Python 3.13 only
+python3_13t - Build for Python 3.13 freethreading only
```

Comment:
I'm assuming we're deliberately not masking this, even though it would be nice to, so that we get the benefit of depgraph checking?

Reply:
I didn't even think of masking it. Why would we? It will be only enabled on packages that were explicitly tested to work with it.

Reply:
I guess to avoid people using it as their Python and having stuff be horribly broken.